omero/developers/Server/LightAdmins.txt

```diff
@@ -0,0 +1,244 @@
+The server's view of administrator restrictions
+===============================================
+
+OMERO 5.4 introduced the concept of a restricted administrator. These
+are generally more powerful than normal OMERO users and group owners but
+they do not have all the powers of full administrators such as ``root``
+because they are restricted in specific ways by their
+``Experimenter.config`` property. For that property, a name of
+``AdminPrivilege:Dance`` (just an example!) with a value of ``false``
+indicates that the user's ability to dance is not that of a full
```

Comment:
From here on, I think this needs a reformulation, this sentence is too long. Genuinely do not understand it. Nothing wrong with the ``dance``, but the further discerning of the ``ability``, between ``administrator`` (restricted administrator or full, hypothetical or real ?) and  ``that of a normal user`` is too big an ask.